exporter/mezmoexporter/exporter_test.go

```diff
@@ -96,54 +96,88 @@ func createSizedPayloadLogData(payloadSize int) plog.Logs {
 	return logs
 }
 
-func TestLogsExporter(t *testing.T) {
-	// Spin up a HTTP server to receive the test exporters...
-	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		assert.Equal(t, "application/json", r.Header.Get("Content-Type"))
-		assert.Equal(t, "mezmo-otel-exporter/"+buildInfo.Version, r.Header.Get("User-Agent"))
+type testServer struct {
+	Instance *httptest.Server
+	Url      string
```

Comment:
```suggestion
	instance *httptest.Server
	url      string
```
nit: keep these unexported

Reply:
Corrected.